src/Compilers/CSharp/Test/Syntax/Diagnostics/DiagnosticTest.cs

```diff
@@ -2517,5 +2520,33 @@ public void Bug9288_keycontainer()
         // get to the third case (but there's still error handling if that changes)
 
         #endregion
+
+        #region PathMap Linux Tests
+        // Like the above (CoreCLR Signing Tests), these aren't actually syntax tests, but this is in one of only two assemblies tested on linux
+
+        [Fact]
+        public void PathMapKeepsCrossPlatformRoot()
```

Comment:
I'm not sure how to do that, but can we also test the platform-specific case (when no slashes can be detected)?